src/nav/nav.css

```diff
@@ -29,4 +29,132 @@
     display: flex;
     gap: 45px;
     padding-right: 100px;
-}
\ No newline at end of file
+}
+
+.navig_item {
+    border: 0;
+    background: none;
+    cursor: pointer;
+}
+
+.modal_top_line {
```

Comment:
I mentioned in another PR that it's better not to name things according to their position. better think about its semantic meaning